tests/ProxyManagerTest/Factory/LazyLoadingGhostFactoryTest.php

```diff
@@ -102,6 +102,39 @@ public function testWithOptionalFactory()
     }
 
     /**
+     * @dataProvider skipPropertiesFixture
+     *
+     * @param $className
+     * @param $propertyName
+     * @param $properties
+     */
+    public function testSkipProperties($className, $propertyName, $properties)
+    {
+        $factory      = new LazyLoadingGhostFactory();
+        $ghostObject  = $factory->createProxy(
+            $className,
+            function () use ($propertyName) {
+                $this->fail(sprintf('The property %s not need to be lazy loaded', $propertyName));
```

Comment:
The error message should probably be `The Property "%s" was not expected to be lazy-loaded`